packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -692,4 +707,61 @@ export class SDKClient {
 
     return fileId;
   };
+
+  /**
+   * @dev Deletes `fileId` file from the Hedera Network utilizing Hashgraph SDK client
+   * @param client
+   * @param fileId
+   * @param requestId
+   * @param callerName
+   * @param interactingEntity
+   */
+  private deleteFile = async (
+    client: Client,
+    fileId: FileId,
+    requestId?: string,
+    callerName?: string,
+    interactingEntity?: string,
+  ) => {
+    // format request ID msg
+    const requestIdPrefix = formatRequestIdMessage(requestId);
+
+    try {
+      // Create fileDeleteTx
+      const fileDeleteTx = await new FileDeleteTransaction()
+        .setFileId(fileId)
+        .setMaxTransactionFee(
```

Comment:
Q: what's this calculation here?
We should have to calculate it since it's a transaction initiated by us.
Maybe just set a reasonable limit based on testing.
Also see https://testnet.mirrornode.hedera.com/api/v1/docs/#/network/getNetworkFees, FileDelete fees is not supported by mirror node

Reply:
I have revert back to a max amount of 2 HBARs for this method.